src/main/java/io/github/vhoyon/bot/commands/CommandMusicPlay.java

```diff
@@ -71,24 +72,32 @@ public void action(){
 		if(!isConnectedToVoiceChannelMember()){
 			new BotError(this, lang("NotConnected"), true);
 		}
+		else if(this.isPlaying()
+				&& this.getConnectedVoiceChannelBot() != this
```

Comment:
Did you test this? Just by looking at the code, I feel like this won't work because you used `!=` instead of `![...].equals()` here.

Java likes to tests references, which is not what we seek for here ;)

Reply:
Yeah i tested it

Reply:
Oh, interesting, but that probably means that the references in memory are the same... very interesting.

Just to make sure though, can you change it to `![...].equals()`? a general rule of thumbs in java is that when you want to test for equality with objects, always use `.equals()` because you never know how the JVM will handle memory. Just to make sure this code is future proof ya know!

Also, when you test stuff, make sure to remove the label `!Requires: Testing` and add `*verified`, so the "did you test it?" question is not relevant anymore!